internal/event/client.go

```diff
@@ -0,0 +1,54 @@
+package event
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/eventbridge"
+	"github.com/aws/aws-sdk-go-v2/service/eventbridge/types"
+)
+
+const source = "opg.poas.lpastore"
+
+type EventBridgeClient interface {
+	PutEvents(ctx context.Context, params *eventbridge.PutEventsInput, optFns ...func(*eventbridge.Options)) (*eventbridge.PutEventsOutput, error)
+}
+
+type Client struct {
+	eventBusName string
+	svc          EventBridgeClient
+}
+
+func NewClient(cfg aws.Config, eventBusName string) *Client {
+	return &Client{
+		svc: eventbridge.NewFromConfig(cfg, func (o *eventbridge.Options) {
+			o.BaseEndpoint = aws.String(os.Getenv("AWS_EVENTBRIDGE_ENDPOINT"))
```

Comment:
this `os.Getenv` could be extracted to a param, then you would be handling all of the envvars at the top level

Reply:
I was thinking that if it was in the client itself, it wouldn't have to be duplicated in each of the lambdas when we create the client. Is it a general good practice to have env vars at the top level? I'm curious of trade offs that I haven't thought of

Reply:
I personally prefer having `os.Getenv`s grouped in `main` (or whatever the top thing is), because then it is obvious how to control the values in tests. Saying that, the aws packages pull values whenever they want, so if it is easier here then feel free to leave it